package.json

```diff
@@ -55,15 +55,16 @@
     "codecov": "^1.0.1",
     "concurrently": "^2.0.0",
     "dirty-chai": "^1.2.2",
-    "eslint": "^1.10.3",
-    "eslint-config-airbnb": "^5.0.1",
-    "eslint-plugin-react": "^3.16.1",
+    "eslint": "^2.9.0",
+    "eslint-config-airbnb": "^9.0.1",
+    "eslint-plugin-import": "^1.8.0",
+    "eslint-plugin-react": "^5.1.1",
     "mocha": "^2.4.5",
     "nyc": "^6.4.0",
     "rimraf": "^2.5.2",
     "sinon": "^1.17.3",
     "sinon-chai": "^2.8.0",
-    "watch": "^0.17.1"
+    "watch": "^0.18.0"
```

Comment:
I'm a contributor here now as well.  Good news, the watcher is way faster.  We still feel some delay because the linter is slow to run.  Hope to make more helpful updates to the watch package in the future.